docs/docker_container.md

```diff
@@ -73,7 +76,7 @@ You can build your own Docker image executing the `make docker_build` command in
 In the `./dist` directory it will generate: 
 
 - image tagged as openvino/model_server:latest - with CPU, NCS, and HDDL support
-- image tagged as openvino/model_server-gpu:latest - with CPU, NCS, HDDL, and iGPU support
+- image tagged as openvino/model_server:latest-gpu - with CPU, NCS, HDDL, and iGPU support
```

Comment:
Why such change? `model_server-gpu:latest` should exist as well

Reply:
It's because openvino/model_server:latest-gpu is used later in docker run command. But I can change it back..

Reply:
> 

If its used later in docker run command then its good to stay with `model_server:latest-gpu`